prometheus-ksonnet/grafana/config.libsonnet

```diff
@@ -1,49 +1,35 @@
 {
   _config+:: {
-    admin_services+:: [
-      {
-        title: 'Grafana (Light)',
-        path: 'grafana',
-        params: '/?search=open&theme=light',
-        url: 'http://grafana.%(namespace)s.svc.%(cluster_dns_suffix)s/' % $._config,
-        allowWebsockets: true,
-      },
-      {
-        title: 'Grafana (Dark)',
-        path: 'grafana',
-        params: '/?search=open&theme=dark',
-        url: 'http://grafana.%(namespace)s.svc.%(cluster_dns_suffix)s/' % $._config,
-        allowWebsockets: true,
-      },
-    ],
-  },
+    // configure Grafana Lib based upon legacy configs below
+    rootUrl: $._config.grafana_root_url,
+    provisioningDir: $._config.grafana_provisioning_dir,
+    dashboardConfigMapCount: $._config.dashboard_config_maps,
 
-  grafana_config:: {
-    sections: {
-      'auth.anonymous': {
-        enabled: true,
-        org_role: 'Admin',
-      },
-      server: {
-        http_port: 3000,
-        router_logging: true,
-        root_url: $._config.grafana_root_url,
-      },
-      analytics: {
-        reporting_enabled: false,
-      },
-      users: {
-        default_theme: 'light',
+    grafana_ini+: {
+      sections+: {
+        'auth.anonymous': {
+          enabled: true,
+          org_role: 'Admin',
+        },
+        explore+: {
+          enabled: true,
+        },
       },
-      explore+: {
-        enabled: true,
-      },
-    },
-  },
 
-  local configMap = $.core.v1.configMap,
+    } + $.grafana_config,  //<--legacy config
+
+    // LEGACY CONFIGS:
+    // Grafana config options.
+    grafana_root_url: '',
+    grafana_provisioning_dir: '/etc/grafana/provisioning',
+    dashboard_config_maps: 8,
+
+    // Optionally add labels to grafana config maps.
+    grafana_dashboard_labels: {},
+    grafana_datasource_labels: {},
+    grafana_notification_channel_labels: {},
```

Comment:
These seem not in use anywhere...

Reply:
yeah, but they are a part of the original API, so we need to support it.

Reply:
I see the issue, fixed.